worlds/animal-well/regions.py

```diff
@@ -350,8 +354,8 @@ class AWData(NamedTuple):
     bear_slink_room: {
         iname.slink:
             AWData(AWType.location),
-        bear_transcendental:  # descend, jump into left wall
-            AWData(AWType.region, [[iname.bubble]]),
+        bear_transcendental:  # descend, jump into left wall, or disc hop from the platforms underneath. TODO: too tight to not be disc_hop_hard? It's close.
+            AWData(AWType.region, [[iname.bubble], [iname.disc_hop]]),
```

Comment:
```suggestion
        bear_transcendental:  # descend, jump into left wall, or disc hop from the platforms underneath. TODO: too tight to not be disc_hop_hard? It's close.
            AWData(AWType.region, [[iname.slink, iname.bubble], [iname.top, iname.bubble], [iname.slink, iname.disc_hop], [iname.top, iname.disc_hop]]),  # todo: fix formatting, check if you can use ball reliably
```

Need slink or top to actually open the door and leave the region that direction. Maybe ball?